### ISSUE TITLE
Dependency-Track returns 500.

### ISSUE THREAD

**Issue by morganmccarley** [DependencyTrack/dependency-track#621]
The defect may already be reported! Please search for the defect before creating one.

### Current Behavior:
When I navigate to a project, https://[URL]/dependency-track/project/?uuid=[UUID], and wait a minute, I will get a pop-up window with the following "An unexpected error occurred. Please contact the Dependency-Track administrator for assistance (500)."  
There is nothing in the tomcat or Dependency-Track logs (using the default logging levels).  
Using the browser developer tools it looks as if the following API call is the issue: https://[URL]/dependency-track/api/v1/finding/project/[UUID]?searchText=&sortOrder=asc
The response from this request is "{"message": "An error occurred. Please try again."}
Also, the Overview and Dependencies tabs seem to be populated just fine, but the Audit tab is empty.

### Steps to Reproduce:
See Above.

### Environment:
  
  - Dependency-Track Version: 3.7.1
  - Distribution: Traditional WAR
  - BOM Format & Version: Cyclone DX, most current.
  - Database Server: MySQL - AWS RDS.
  - Browser: IE, Chrome, Mozilla.

### Additional Details:
I can provide screen shots if necessary.
This began occurring after an upgrade to the latest version and a move from the H2 DB to an AWS MySQL instance.

(e.g. detailed explanation, stacktraces, related issues, suggestions how to fix, links for us to have context, eg. stackoverflow, gitter, etc)

**Comment by stevespringett** [comment]
The current UI is being replaced by a new frontend in Dependency-Track v3.8 which will likely be released in the next week or two. If there's an issue with the old UI, it will not be fixed. All work is focued on the new frontend.

Also, it's not possible to provide support for the traditional or executable WAR directly. There are too many environment variables which may contribute to errors. To remove the majority of environmental issues, the Docker container is the preferred method of deployment. 

If you're getting the same issues with the container, then I'd look at the database. Only very specific configurations of MySQL are supported. Anything outside of that will not work properly. Most folks use either PostgreSQL or Microsoft SQL Server. AWS has a Postgres option.

**Comment by morganmccarley** [comment]
thank you Steve, I will do some testing.  Are you talking about the MySQL configurations here - https://docs.dependencytrack.org/getting-started/database-support/?  This could be the problem, if I follow these instructions does MySQL typically work?

**Comment by stevespringett** [comment]
Correct. If you follow the instructions on that page and ensure you're not using a UTF-8 collation, then it should work fine.

If all this is one, then yes, MySQL does work fine.
